File: Tools/codetesters/pytesters/getgene.py
#!/usr/bin/env python

import ensembl

serverRegistry = ensembl.get_registry(host='ensembldb.ensembl.org', user='anonymous')

coreDBAdaptor = serverRegistry.get_DBAdaptor('homo_sapiens', 'core', '47_36i')

geneAdaptor = coreDBAdaptor.get_adaptor('gene')

geneAdaptor.fetch_by_stable_id("

File: Tools/codetesters/perltesters/get_gene.pl
#!/usr/bin/perl

use Bio::EnsEMBL::Registry;

sub fetch_by_gene_id {
	my ($stable_id, $parameters) = @_;

	my ($host, $user) = ($parameters->{host}, $parameters->{user});

	my $registry = 'Bio::EnsEMBL::Registry';

	$registry->load_registry_from_db(
		-host => $host,
		-user => $user
	);
	
	my $gene_adaptor = $registry->get_adaptor( 'Mouse', 'Core', 'Gene' );
	my $gene = $gene_adaptor->fetch_by_display_label($stable_id);
	
	return $gene;
	
}

my $parameters;

$parameters->{"host"} = "ensembldb.ensembl.org";
$parameters->{"user"} = "anonymous";

# scalar, because it's a reference
my $attributes;

$attributes = fetch_by_gene_id("IL13", $parameters)->get_all_Attributes();

foreach $attr (@$attributes)
{
	print $attr->name();
}
